documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md

```diff
@@ -92,17 +92,15 @@ In particular, it would be helpful to review our approach to the following:
     - resources reach a certain usage threshold or are auto-scaled
   - Logging capabilities (e.g. search, filtering) and retention policies to assist with troubleshooting
 - **Procedures**
-  - Internally-defined service level agreements for availability and incident response
   - Documented incident response plan with steps and roles identified
   - Documented plan for receiving and responding to alerts
   - Staff trainings, tabletop and/or dry-runs for common types of incidents
 
 ### Communication strategy
 
-The two main communication outputs needed for this deliverable are:
+The two main communication outputs needed for this deliverable is an email campaign that is sent to the people with whom we plan to share the site.
```

Comment:
nit: `The two main communication outputs` should now be `The main communication output'